### PR TITLE
remove NSX references from Crowbar deployment (SOC-10081)

### PR DESCRIPTION
**Pull request by csymons-suse**
NSX on SOC 7 and 8 Crowbar was not tested or qualified.

make plug-in consistent with hyphen (SOC-10081)
plug-in is the format specified in the Style Guide

(cherry picked from commit 23c491cfb57a5e237b5bf9f80e035137b5fcf876)

### CODE REVIEW

File: xml/depl_nodes.xml [hunk 1]
```diff
@@ -6120,6 +6075,231 @@ monasca-installer on the &crow; node</guimenu>
...
+  </bridgehead>
+  <variablelist>
+   <varlistentry>
+    <term>
+     monitor_libvirt
+    </term>
+    <listitem>
+     <para>
+      The global switch for toggling libvirt monitoring. If set to
+      <parameter>true</parameter>, libvirt metrics will be gathered on all
```
Comment (asettle) [inline xml/depl_nodes.xml]:
```suggestion
<parameter>true</parameter>, libvirt metrics are gathered on all
```

File: xml/depl_nodes.xml [hunk 2]
```diff
@@ -6120,6 +6075,231 @@ monasca-installer on the &crow; node</guimenu>
...
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     monitor_ceph
+    </term>
+    <listitem>
+     <para>
+      The global switch for toggling &ceph; monitoring. If set to
+      <parameter>true</parameter>, &ceph; metrics will be gathered on all
```
Comment (asettle) [inline xml/depl_nodes.xml]:
```suggestion
<parameter>true</parameter>, &ceph; metrics are gathered on all
```

File: xml/depl_nodes.xml [hunk 3]
```diff
@@ -6120,6 +6075,231 @@ monasca-installer on the &crow; node</guimenu>
...
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     cache_dir
+    </term>
+    <listitem>
+     <para>
+      The directory where monasca-agent will locally cache various metadata
```
Comment (asettle) [inline xml/depl_nodes.xml]:
```suggestion
The directory where <literal>monasca-agent</literal> locally caches various metadata
```

File: xml/depl_nodes.xml [hunk 4]
```diff
@@ -6120,6 +6075,231 @@ monasca-installer on the &crow; node</guimenu>
...
+   <varlistentry>
+    <term>
+     disk_collection_period
+    </term>
+    <listitem>
+     <para>
+      Specifies a minimum interval in seconds for collecting disk metrics.
+      Increase this value to reduce I/O load. If the value is lower than the
+      global agent collection period (<option>check_frequency</option>), it
+      will be ignored in favor of the global collection period.
```
Comment (asettle) [inline xml/depl_nodes.xml]:
```suggestion
is ignored in favor of the global collection period.
 ```

File: xml/depl_nodes.xml [hunk 5]
```diff
@@ -6120,6 +6075,231 @@ monasca-installer on the &crow; node</guimenu>
...
+    </term>
+    <listitem>
+     <para>
+      Specifies the number of ping command processes to run concurrently when
+      determining whether the VM is reachable. This should be set to a value
+      that allows the plug-in to finish within the agent's collection period,
+      even if there is a networking issue. For example, if the expected number
+      of VMs per &compnode; is 40 and each VM has one IP address, then the
+      plug-in will take at least 40 seconds to do the ping checks in the
+      worst-case scenario where all pings fail (assuming the default timeout of
```
Comment (asettle) [inline xml/depl_nodes.xml]: Lines 6152 - 6155 is confusing to read. I thought it was a run on sentence at first but I realise the break. I wonder if this sounds any better:

"For example, if the expected number of VMs per &compnode; is 40 and each VM has one IP address, then the plug-in takes at least 40 seconds to perform ping checks in a worst-case scenario where all pings fail (assuming the default timeout of 1 second)."

Need other eyes, me thinks. Maybe I'm just tired? @dmpop

File: xml/depl_nodes.xml [hunk 6]
```diff
@@ -6120,6 +6075,231 @@ monasca-installer on the &crow; node</guimenu>
...
+   <varlistentry>
+    <term>
+     nova_refresh
+    </term>
+    <listitem>
+     <para>
+      Specifies the number of seconds between calls to the &o_comp; API to
+      refresh the instance cache. This is helpful for updating VM hostname and
+      pruning deleted instances from the cache. By default, it is set to 14,400
+      seconds (four hours). Set to 0 to refresh every time the Collector runs,
```
Comment (asettle) [inline xml/depl_nodes.xml]: collector with a capital?

File: xml/depl_nodes.xml [hunk 7]
```diff
@@ -6120,6 +6075,231 @@ monasca-installer on the &crow; node</guimenu>
...
+      or to <parameter>None</parameter> to disable regular refreshes entirely.
+      In this case, the instance cache will only be refreshed when a new
```
Comment (asettle) [inline xml/depl_nodes.xml]:
```suggestion
In this case, the instance cache is only refreshed when a new
```

File: xml/depl_nodes.xml [hunk 8]
```diff
@@ -6120,6 +6075,231 @@ monasca-installer on the &crow; node</guimenu>
...
+   <varlistentry>
+    <term>
+     vnic_collection_period
+    </term>
+    <listitem>
+     <para>
+      Specifies a minimum interval in seconds for collecting disk metrics.
+      Increase this value to reduce I/O load. If the value is lower than the
+      global agent collection period (<option>check_frequency</option>), it
+      will be ignored in favor of the global collection period.
```
Comment (asettle) [inline xml/depl_nodes.xml]:
```suggestion
is ignored in favor of the global collection period.
```

File: xml/depl_nodes.xml [hunk 9]
```diff
@@ -6120,6 +6075,231 @@ monasca-installer on the &crow; node</guimenu>
...
+   <varlistentry>
+    <term>
+     vnic_collection_period
+    </term>
+    <listitem>
+     <para>
+      Specifies a minimum interval in seconds for collecting VM network
+      metrics. Increase this value to reduce I/O load. If the value is lower
+      than the global agent collection period
+      (<parameter>check_frequency</parameter>), it will be ignored in favor of
```
Comment (asettle) [inline xml/depl_nodes.xml]:
```suggestion
(<parameter>check_frequency</parameter>), it is ignored in favor of
```

Comment (csymons-suse) [reply xml/depl_nodes.xml]: 1. For this and several other instances of `will`...These are conditional sentences, and `will` is appropriate.

**Pull request merged**